File: src/Auth/Auth.Shared/DTOs/v1/UserDto.cs
namespace Dyvenix.App1.Auth.Shared.DTOs.v1;

public class UserDto
{
	public string Id { get; set; } = null!;
	public string UserName { get; set; } = null!;
	public string Email { get; set; } = null!;
	public string PhoneNumber { get; set; } = null!;
	public bool EmailConfirmed { get; set; }
	public bool LockoutEnabled { get; set; }
	public DateTimeOffset? LockoutEnd { get; set; }
}

File: src/UI/Angular/src/app/core/services/auth/dto/user.dto.ts
export interface UserDto {
    id: string;
    tenantId: string;
    userName: string;
    email: string;
    phoneNumber: string | null;
    emailConfirmed: boolean;
    lockoutEnabled: boolean;
    lockoutEnd: string | null;
    twoFactorEnabled: boolean;
}

export interface UserSummaryDto {
    id: string;
    tenantId: string;
    userName: string;
    email: string;
}
